styles/styles.css

```diff
@@ -215,27 +235,31 @@ form {
   height: 40px;
 }
 
-h2 {
+.offers__title {
   color: #383838;
   font-size: 40px;
   font-weight: var(--font-medium);
-  margin-top: 120px;
-  margin-bottom: 20px;
+  margin-bottom: 104px;
+  position: relative;
 }
 
-.line {
+.offers__title:after {
+  display: inline-block;
+  content: "";
   width: 100px;
   height: 4px;
   background-color: #f5bd41;
-  margin-bottom: 80px;
-  margin-left: auto;
-  margin-right: auto;
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translate(-50%, 500%);
 }
 
 .offers {
+  padding: 0;
   margin-bottom: 120px;
   display: flex;
-  justify-content: space-evenly;
+  justify-content: space-around;
```

Comment:
use space-between, also better to use container  which could be restricted by max-width like width on design and had property width:100%